### ISSUE TITLE
Can't translate the pages

### ISSUE THREAD

**Issue by Hambern** [wintercms/wn-pages-plugin#26]
- Winter CMS Build: 1.2.1 
- Pages Plugin Version: 2.0
- PHP Version: 8.2

### Description:
I'm trying to get this to work with Winter.Translate (v.2.1). But I can not get them to work together. The only field I can translate is the url-field, nothing else. And when I use already translated pages from an old installation, it can't shift between the languages.

### Steps To Reproduce:
Install Winter.Translate and Winter.Pages via composer and then try to translate the fields in the pages.

**Comment by mjauvin** [comment]
You must use Winter.Pages version 2.0.2.

**Comment by Hambern** [comment]
Well... I do... But I upgraded from the Rainlab versions of the plugins. Could that be the problem?

**Comment by mjauvin** [comment]
Did you run the migrations?

`php artisan migrate`

**Comment by Hambern** [comment]
Yes. I've even reinstalled the both plugins several times. No luck. Regular textfields is translatable. Lite when you add buttons to the menu, but not the big "content"-field or the "title"-field.

**Comment by mjauvin** [comment]
I can't reproduce this.

Can you try this from a fresh install of winter cms and plugins? Also, don't include  plugins other than translate and pages.

**Comment by mjauvin** [comment]
Do you see any errors in the JavaScript console when you save the page (first or second time) ?

**Comment by mjauvin** [comment]
And if you create a new page?

**Comment by mjauvin** [comment]
Ok.

But you said the bug didn't show up the second time around. I wanted to know if this is true for any NEW page you create.

**Comment by mjauvin** [comment]
Can you please try version 2.0.2 and confirm the bug does not trigger?

**Comment by mjauvin** [comment]
@LukeTowers from my testing, something got broken with version 2.1.0 (worked fine up to 2.0.3)

**Comment by LukeTowers** [comment]
We've identified the issue and I'm working on a fix right now

**Comment by ivang76** [comment]
Hi @LukeTowers, have you an idea of a timeline for this fix maybe? I found it as well and at the moment I just forced the downgrade but the editors missed some contents and they have overwritten urls, so you can imagine ;-)
thank you!

**Comment by ivang76** [comment]
any updates?
thank you

**Comment by LukeTowers** [comment]
Still working on the fix, just finished the improvements to the four plugins (Translate, Sitemap, Pages, & Blog) that will allow for multilingual sitemaps so I'm moving back to fixing this issue now.

**Comment by mjauvin** [comment]
@ivang76 @hambern @brutalhost, if you can test [this fix](#33) in your environment, that'd be nice!

**Comment by der-On** [comment]
Thanks for fixing this! :bow:

**Comment by ivang76** [comment]
@mjauvin sorry for the late and thanks a lot for the fix. I'm using the latest version on different sites and everything seems work correctly!
just one question... I don't remember if there was some artisan command or similar for this, but I see that the versions in the backend system/updates are different from the ones managed via composer. How can I keep them updated?

**Comment by mjauvin** [comment]
@ivang76 the backend system/updates is non functional for now. Once the marketplace is released, it will be possible to use either that or composer to manage plugins/themes.
